server/models/match.rb

```diff
@@ -4,7 +4,12 @@ class Match < Sequel::Model
   # TODO i think this should end with "es" since its a *_to_many
   one_to_many :user_season_match, :class => :UserSeasonMatch
   def validate
-    validates_presence :scheduled_for
+    validates_integer :best_of
+    validates_presence [:scheduled_for, :best_of]
```

Comment:
Whoops! Should have added this when I added the schema changes. I'll make sure I don't make that mistake again.